css/css-values/getComputedStyle-calc-mixed-units-003.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+
+  <meta charset="UTF-8">
+
+  <title>CSS Values Test: computed value of calc() values using multiplication/division with mixed units</title>
+
+  <link rel="author" title="Gérard Talbot" href="http://www.gtalbot.org/BrowserBugsSection/css21testsuite/">
```

Comment:
Oops, please remove this or put your own name in :) (If you can't decide, I don't put my name in, I like minimal tests.)

Reply:
Whoops copy pasta haha — thanks :)

Resolved in [2925a74](https://github.com/web-platform-tests/wpt/pull/42517/commits/2925a74b2be2eafa9fb0bef893287d02fee839a6)